src/appshell/qml/DevTools/Interactive/InteractiveTests.qml

```diff
@@ -119,10 +119,24 @@ Rectangle {
             onClicked: testModel.openWidgetDialog()
         }
 
+        PopupView {
+            id: popup
+
+            Rectangle {
+                color: "red"
+                width: 300
+                height: 300
+            }
+        }
+
         FlatButton {
             width: 200
-            text: "Widget dialog async"
-            onClicked: testModel.openWidgetDialogAsync()
+            text: "Widget dialogg async"
+            onClicked: popup.show()//testModel.openWidgetDialogAsync()
```

Comment:
Please, remove this comment

Reply:
no, needed to revert changes in this file

Reply:
yeah, forgotten test code, thx